submissions/Yasya23/tiny-js-oop/index.js

```diff
@@ -0,0 +1,61 @@
+class Inhabitants {
+  constructor(species, name, gender) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+  }
+
+  showProperties() {
+    return `${this.species}; ${this.name}; ${this.gender}`;
+  }
+}
+
+class Human extends Inhabitants {
+  constructor(name, gender, saying) {
+    super("human", name, gender);
+    this.hands = 2;
+    this.legs = 2;
+    this.saying = saying;
+  }
+
+  showProperties() {
+    return `${super.showProperties()}; ${this.hands}; ${this.legs}; ${
+      this.saying
+    }`;
+  }
+}
+
+class Animal extends Inhabitants {
+  constructor(species, name, gender, saying) {
+    super(species, name, gender);
+    this.paws = 4;
+    this.saying = saying;
```

Comment:
What makes `saying` different from `name` in terms of generalization into the base class?

Reply:
Hello, thanks for checking! There is no difference in these cases, fixed it.